dandiapi/api/migrations/0005_default_oauth_application.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.0.9 on 2020-12-17 23:59
+# flake8: noqa N806
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def create_application(apps, schema_editor):
+    Application = apps.get_model('oauth2_provider', 'Application')
+    name = 'Frontend Provider'
```

Comment:
```suggestion
    name = 'DANDI GUI'
```

This seems like a little more descriptive name. Feel free to pick something else, if we have a better term for the Vue-based frontend app, but using the word "Provider" here is misleading.

Reply:
I just spitballed that, I don't have strong opinions on the name.